CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.12.28+1
```

Comment:
This doesn't match the change in the pubspec version

Reply:
That's intentional—`0.12.28+1-dev` isn't a valid semantic version, so we use the next closest option, `0.12.29-dev`.

Reply:
Do we intend to publish in this state?
If not, why do we need a valid semantic version?

Reply:
No, the point of the `-dev` flag is to indicate that we *don't* intend to publish. However, pub will still parse the version number, and it may even be relevant for version solving if users have a git dependency on this package.

Reply:
I missed this the first time around. What is preventing us from publishing?

Reply:
I'm just waiting for a few more changes to land so that we can publish them all together.

Reply:
sg